crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -327,3 +342,52 @@ pub struct EthContractAddresses {
     #[serde(rename = "GpsStatementVerifier")]
     pub gps_statement_verifier: EthereumAddress,
 }
+
+#[cfg(test)]
+mod tests {
+    use super::transaction::Event;
+    use crate::core::{ContractAddress, EventData, EventKey};
+    use pedersen::StarkHash;
+
+    #[test]
+    fn event_data_serialization() {
+        let event = Event {
+            from_address: ContractAddress::from_hex_str(
+                "0x06fbd460228d843b7fbef670ff15607bf72e19fa94de21e29811ada167b4ca39",
```

Comment:
Ah, this part has changed since your review. No, we don't need a real contract address. We don't even need the Event anymore for testing data serialization.